go.mod

```diff
@@ -5,14 +5,20 @@ go 1.13
 require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
 	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/getsentry/sentry-go v0.5.1
 	github.com/gin-gonic/gin v1.5.0
+	github.com/go-openapi/spec v0.19.6 // indirect
+	github.com/go-openapi/swag v0.19.7 // indirect
 	github.com/go-redis/redis v6.15.6+incompatible
+	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/spf13/viper v1.6.2
 	github.com/stretchr/testify v1.5.1
 	github.com/swaggo/gin-swagger v1.2.0
 	github.com/swaggo/swag v1.6.5
 	github.com/trustwallet/blockatlas v1.0.37
 	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5 // indirect
+	golang.org/x/net v0.0.0-20200301022130-244492dfa37a // indirect
+	golang.org/x/tools v0.0.0-20200303225724-c5a141475315 // indirect
```

Comment:
it's not clear what these new dependencies are needed for. Mind explaining?

Reply:
Populated after `make install`, not sure why

Reply:
Sounds like we don't need them though?

Reply:
Removed, not sure why go does it.

Reply:
ok